### PR TITLE
configurable email subject

### PR DESCRIPTION
**Pull request by marcinapostoluk**
Notebooker already had an option to specify email subject but it was not possible to configure it via the UI. Now added and also preserving it in the results for re-runs

### CODE REVIEW

**Review by jonbannister — COMMENTED** [review]
Looks good, but I think there is a small bug in the scheduler config - it doesn't seem to be saving the new field 
image

File: notebooker/utils/notebook_execution.py [hunk 1]
```diff
@@ -24,7 +24,11 @@ def send_result_email(result: Union[NotebookResultComplete, NotebookResultError]
     report_title = (
         result.report_title.decode("utf-8") if isinstance(result.report_title, bytes) else result.report_title
     )
-    subject = result.email_subject or f"Notebooker: {report_title} report completed with status: {result.status.value}"
+    default_subject = f"Notebooker: {report_title} report completed with status: {result.status.value}"
+    if isinstance(result, NotebookResultComplete):
+        subject = result.email_subject or default_subject
+    else:
+        subject = default_subject
```
Comment (danyaalm) [inline notebooker/utils/notebook_execution.py:31]:
```suggestion
    subject = f"Notebooker: {report_title} report completed with status: {result.status.value}"
    if isinstance(result, NotebookResultComplete):
        subject = result.email_subject or subject
```

Comment (marcinapostoluk) [reply notebooker/utils/notebook_execution.py]: thanks, now changed

**Comment by marcinapostoluk** [comment]
> Looks good, but I think there is a small bug in the scheduler config - it doesn't seem to be saving the new field image

hmmm re-tested and does work fine for me :/

**Pull request merged**